legal-api/src/legal_api/services/search_service.py

```diff
@@ -269,3 +269,32 @@ def get_search_filtered_filings_results(business_json,
                 draft_results.append(draft)
 
         return draft_results
+
+    @staticmethod
+    def get_affiliation_mapping_results(org_id: int = None):
+        """Return affiliation mapping results for the given organization ID."""
+        query = db.session.query(
+            Filing.id.label('filing_id'),
+            Business.identifier,
+            Filing
+            .filing_json['filing'][Filing._filing_type]['nameRequest']['nrNumber']  # pylint: disable=protected-access
```

Comment:
when i was testing things with the raw sql, it would filter out numbered companies because there was no `nrNumber` value.  Can you confirm this doesn't happen?

Reply:
interesting. ideally this should only be in select query